packages/react/src/VideoPlayer/VideoPlayer.module.css

```diff
@@ -87,6 +93,13 @@
   color: var(--brand-videoPlayer-playButton-fgColor-rest);
 }
 
+@media (forced-colors: active) {
+  .VideoPlayer__playButton {
+    color: Highlight;
```

Comment:
The coners look noticeably different to the other adjacent buttons that have rounded corners, could we add them here too?

Reply:
I added rounding to the Range input focus states too, but I can remove that if desired 👍